shared/fs/common/path-item-action.js

```diff
@@ -15,7 +15,8 @@ import {
   iconCastPlatformStyles,
   type OverlayParentProps,
 } from '../../common-adapters'
-import {PathItemIcon, PathItemInfo} from '../common'
```

Comment:
did you want to remove them from `fs/common.js` as well?

Reply:
They are still supposed to be exported there; this is just because it'd cause circular dependency warning in console.